### PR TITLE
docs: add MAINTAINING.md for maintainers guide

### PR DESCRIPTION
**Pull request by dhmlau**
Add `MAINTAINING.md` for maintainers' guide. 

Some of the contents are coming from the email content when we're inviting maintainers, regarding the benefits, expectation and responsibilities.  

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

### CODE REVIEW

**Comment by raymondfeng** [comment]
Should we also create `loopback-next/MAINTAINERS` to list current maintainers?

**Comment by dhmlau** [comment]
@raymondfeng , similar to https://github.com/nodejs/node/blob/master/deps/uv/MAINTAINERS.md?  Are you referring to adding another file or merge the 2 content into one file?

**Comment by raymondfeng** [comment]
Yes, as two separate files. `MAINTAINERS` is only in the github repo.

**Comment by dhmlau** [comment]
Added the list of maintainers in the last commit.  However, I have a question.  Looking at the members in https://github.com/orgs/strongloop/teams/loopback-maintainers/members:
- not all the core maintainers are there
- I don't recall we've invited some community maintainers to be LB4 maintainers. 

The description of this github team seems to meant to be for LB4 maintainers.  @bajtos, maybe you have more info on that? Thanks.

File: docs/MAINTAINING.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  http://loopback.io/doc/en/contrib/triaging-pull-requests.html and
+  http://loopback.io/doc/en/contrib/style-guide.html (but feel free to propose
+  changes to these conventions!)
+- To avoid possible confusion, we don't have any specific requirements about the
+  amount of time you should spend on the project - just keep working on things
+  that you find important to you, in a pace that suits you. We may ask you to
+  take a look at issues and pull requests related to code you contributed
+  yourself.
+
+## More questions?
```
Comment (nabdelgadir) [inline docs/MAINTAINING.md]: If someone has a question that isn't covered here, should we also include something like "If you have more questions, feel free to reach out by * _insert way for them to reach us_ *"? Also, how do non-IBM maintainers communicate with the team outside GitHub issues/PRs?

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- Diana Lau ([@dhmlau](https://github.com/dhmlau))
+- Dominique Emond ([@emonddr](https://github.com/emonddr))
+- Hugo Da Roit ([@Yaty](https://github.com/Yaty))
+- Janny Hou ([@jannyhou](https://github.com/jannyhou))
+- Mario Estrada ([@marioestradarosa](https://github.com/marioestradarosa))
+- Miroslav Bajtos ([@bajtos](https://github.com/bajtos))
+- Nora Abdelgadir ([@nabdelgadir](https://github.com/nabdelgadir))
+- Raymond Feng ([@raymondfeng](https://github.com/raymondfeng))
+- TN ([@thinusn](https://github.com/thinusn))
+- Yaapa Hage ([@hacksparrow](https://github.com/hacksparrow))
```
Comment (raymondfeng) [inline MAINTAINERS.md]: Maybe we should divide the list into IBMers and community members. Or mark us as IBMers.

File: docs/MAINTAINING.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+Congratulations! Since you're reading this page, you are probably already a
+maintainer of this repo or close to be one. Thank you for your contributions so
+far!
+
+## Why do I want to be a maintainer?
+
+- Greater influence on LoopBack's future direction
+- Commit (write) rights to `loopback-next` repo
+- Ability to review and land pull requests, edit/categorize/close issues
+- Access to a private Gitter room for LoopBack maintainers
```
Comment (raymondfeng) [inline docs/MAINTAINING.md]: Gitter or Slack?

File: MAINTAINERS.md (hunk 2 above)
Comment (dhmlau) [reply MAINTAINERS.md]: How about `core maintainers` and `community maintainers` that are used somewhere else in the docs? e.g. https://loopback.io/doc/en/contrib/Governance.html.

File: docs/MAINTAINING.md (hunk 3 above)
Comment (dhmlau) [reply docs/MAINTAINING.md]: Actually, I meant to remove it.  Thanks for catching that.

File: docs/MAINTAINING.md (hunk 1 above)
Comment (dhmlau) [reply docs/MAINTAINING.md]: That's a good question! I guess the simplest way is to create a github issue.  Another way is to email the maintainers (email might be found in some of our github profiles), but I'd rather not encouraging people to email us. :) 

@strongloop/loopback-maintainers, any suggestions?

Comment (dhmlau) [reply docs/MAINTAINING.md]: @nabdelgadir, I've added the last "question" regarding your comment. Thanks!

Comment (jannyHou) [reply docs/MAINTAINING.md]: The questions section LGTM 👍

**Comment by dhmlau** [comment]
@jannyHou , you mean this one? https://github.com/strongloop/loopback-next/issues/943.  This is more about the criteria to be a maintainer. :)

File: MAINTAINERS.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+## Project Maintainers
```
Comment (bajtos) [inline MAINTAINERS.md]: This file seems to duplicate the content in our README, see https://github.com/strongloop/loopback-next#team

@dhmlau Please find a way how to have a single place listing the maintainers.

File: docs/MAINTAINING.md (hunk 1 above)
Comment (bajtos) [reply docs/MAINTAINING.md]: FWIW, GitHub offers teams a place for private discussions, see e.g. here: https://github.com/orgs/strongloop/teams/loopback-maintainers/discussions

I am not sure how do notifications work for discussions, using GitHub issues may be a safer way.

**Pull request merged**